src/lib/dnssd/Discovery_ImplPlatform.cpp

```diff
@@ -558,20 +558,18 @@ CHIP_ERROR DiscoveryImplPlatform::Advertise(const CommissionAdvertisingParameter
     {
         PUBLISH_RECORDS(Commissioner);
```

Comment:
This is extremely confusing, because the fact that PUBLISH_RECORDS includes a return is very non-obvious.

Could we please pull the return out of the macro so it's explicit at the return sites?